src/rulesets/rest/2022-05-25/lifecycle-rules.ts

```diff
@@ -106,7 +106,35 @@ const followSunsetRules = new SpecificationRule({
   },
 });
 
+const noNewExperimental = new SpecificationRule({
```

Comment:
How can we feature flag this change? Can we apply a pivot date config here that is read from a github source, so that we can control the rollout based on a the product comms?

Reply:
Is that needed when this is a major version change?